tests/test_peer_did_utils.py

```diff
@@ -50,3 +53,88 @@ def test_encode_service_with_multiple_entries_list():
         "nNvbWVtZWRpYXRvciNzb21la2V5Il19LHsidCI6ImRtIiwicyI6Imh0dHBzOi8vZXhhbXBsZS5jb20vZW5kcG9p"
         "bnQyIiwiciI6WyJkaWQ6ZXhhbXBsZTpzb21lbWVkaWF0b3Ijc29tZWtleTIiXX1d"
     )
+
+
+PEER_DID = "did:peer:0z6MkqRYqQiSgvZQdnBytw86Qbs2ZWUkGv22od935YF4s8M7V"
+
+
+@pytest.mark.parametrize(
+    "test_value",
+    [
+        (
+            "6MkqRYqQiSgvZQdnBytw86Qbs2ZWUkGv22od935YF4s8M7V",
+            "Ed25519VerificationKey2018",
```

Comment:
I guess these tests would have non-good names, makes sense to define ids, they also will improve the input data understanding